### PR TITLE
master-qa-65591

### PR DESCRIPTION
**Pull request by Sharryxue**
https://issues.liferay.com/browse/LRQA-65591
Apply SACRED Model.

### CODE REVIEW

**Comment by Sharryxue** [comment]
NO NEED TESTING.

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 1]
```diff
@@ -2237,28 +2237,16 @@ definition {
 
 	@description = "Log out of Liferay Portal."
 	macro logoutPG {
-		if (isSet(nodePort)) {
-			Navigator.openSpecificURL(url = "http://localhost:${nodePort}");
+		if (isSet(specificURL)) {
+			Navigator.openSpecificURL(url = "${specificURL}");
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/macros/User.macro:2241]: Simple the condition, so it's less different with the [logoutUserPG ](https://github.com/Sharryxue/liferay-portal/pull/254/files#diff-ddf0f6ac34b48653aae4320acee8285c0b63a3a83b2ac227172f7719e491c06eL2255-L2261)macro below, and we can combine them.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase [hunk 2]
```diff
@@ -446,7 +446,7 @@ definition {
 
 		DMDocument.checkoutPG(dmDocumentTitle = "DM Document Title");
 
-		User.logoutUserPG();
+		User.logoutPG();
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase:449]: I remember there was a logout via url macro but fail to find it, in order to apply the SACRED, do you thinks it's better to logout via url(or other better way to logout the user) to save the user logout time?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusergroups/UsergroupsUsecase.testcase [hunk 3]
```diff
@@ -32,47 +32,23 @@ definition {
...
+				userEmailAddress = "userea${userNumber}@liferay.com");
+		}
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Users",
-			panel = "Control Panel",
-			portlet = "User Groups");
+		UserGroup.openUserGroupsAdmin();
 
 		LAR.exportUserGroups(larFileName = "User_Group_Members.lar");
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusergroups/UsergroupsUsecase.testcase:53]: Update this test to add the required data via json as much as possible, only use the UI test steps for the target, so the test can get a real failure info when it fails.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase (hunk 2 above)
Comment (reizero00) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase:449]: I talked to Kenji about this but basically there is a more efficient way to login and logout by just controlling the browser cookies. For logout specifically, as long as we are not testing the logout process, we can implement a way to delete the current cookies instead.

Comment (Sharryxue) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase:449]: Thanks for you reply @reizero00, got it, that would save lots of time just as the way you'd introduced for login. Will wait for it, and try to understand how cookies works with selenium when get a chance.

**Pull request closed without merging**